www/templates/diary/current.html

```diff
@@ -0,0 +1,137 @@
+<!-- <script type="text/javascript" src="file:///Users/AnnaMarie/e-mission-phone/www/js/diary/current.js"></script> -->
+<ion-view ng-class="ionViewBackgroundClass()" title="Current Trip">
+  <ion-nav-bar class="bar-stable">
+  
+  <ion-nav-buttons side="right">
+<button ng-click="diary()" class="button refresh-tiles" >History
+    </button>
+</ion-nav-buttons>
+ </ion-nav-bar>
+     <!--  <a ui-sref="root.main.diary">rrrr</a>
+<!-  <ion-nav-view></ion-nav-view>
+ -->    <ion-content class="has-header" overflow-scroll="true" padding="true">
+
+      <!--   <div style="" class="button-bar">
+            <button class="button button-stable button-block icon ion-refresh"
+                ng-click="refreshMap()">Refresh</button>
+            <!
+            <button style="" class="button button-stable button-block  icon ion-checkmark">Valid</button>
+            -->
+       <!--  </div> -->
+<!-- <a ui-sref="root.main.diary">
+        <button  class="btn btn-danger navbar-btn">
+      back
+    </button>
+    </a> -->
+<!--     <div ng-class="listCardClass()">
+ -->
+        <div class="row" style="margin-top: 20px;">
+
+            <div ng-class="listColLeftClass()">
+                <p style="font-size: 1.2em; line-height: 1em; margin-left: 20px;">Began: {{getdate()}}</p>
+                
+               <!--  <div style="font-size: 15px;white-space: nowrap; width: 0px; color: #999; display: {{ getEarlierOrLater(mapCtrl.geojson.data.properties.start_ts, mapCtrl.geojson.data.id) == ''? 'none':'block';  }}">
+
+                <img src="{{ getEarlierOrLater(mapCtrl.data.properties.start_ts, mapCtrl.data.id)[0] == '-'? 'img/minus.gif':'img/plus.gif';  }}" style="width: 15px; margin-right: 8px;"> 
+
+                {{parseEarlierOrLater(getEarlierOrLater(mapCtrl.geojson.data.properties.start_ts, mapCtrl.geojson.data.id))}} 
+
+                </div> -->
+
+                <!-- <p style="font-size: 1.2em; line-height: 0.6em; margin-left: 20px; margin-top: 5px">{{lat_long()}}</p>
+                <p style="font-size: 1.2em; line-height: 0.6em; margin-left: 20px; margin-top: 5px">{{refresh()}}</p> -->
+                <!-- <script>$scope.refresh(lat(), long());</script>
+                {{refresh(lat(), long())}} -->
+            </div>
+
+            <div ng-class="listColRightClass()">
+
+                <div >
+                            <i class="ion-android-walk" ng-style="brown" style="margin-left: 150px; font-size: 3em"></i>
```

Comment:
You can't hardcode `ion-android-walk`. you need to use the correct icon for the current mode.
`DiaryHelper.getIcon` is your friend.